packages/manager/modules/exchange/src/account/order/account-order.html

```diff
@@ -2,9 +2,10 @@
     <div
         data-wizard
         data-wizard-bread-crumb
-        data-wizard-confirm-button-text="'wizard_pay' | translate"
+        data-wizard-confirm-button-text="ctrl.isOvhtel ? 'wizard_close' : 'wizard_pay' | translate"
```

Comment:
I dont think we can display `Close` as a primary button. We may need a validation from the design team.

Reply:
No it's problem ticket. In fact is close not cancel just for close the modal because no need redirect to order url and as it's wizard it isn't easy to do data-ng-if on step (issue with scope) so i've tried to keep it as simple as possible with a minimum of change / duplication